dmscripts/scan_g_cloud_services_for_bad_words.py

```diff
@@ -122,7 +122,13 @@ def scan_services_for_bad_words(
         framework_slug, 'service_questions_to_scan_for_bad_words', 'service_questions'
     )
 
-    check_services_with_bad_words(
-        output_file_path, framework_slug, client, suppliers, bad_words, questions_to_check, logger, scan_drafts
+    return check_services_with_bad_words(
+        output_file_path,
+        framework_slug,
+        client,
+        suppliers,
+        tuple((bad_word, re.compile(rf"\b{re.escape(bad_word)}\b")) for bad_word in bad_words),
```

Comment:
Rather than changing the signature of the function can we compile the regexes inside it?

Reply:
Only trying to give `scan_services_for_bad_words` more of a reason to exist.